ocaml/xapi/xapi_vbd_helpers.ml

```diff
@@ -95,6 +95,11 @@ let valid_operations ~expensive_sharing_checks ~__context record _ref' : table =
     set_errors Api_errors.other_operation_in_progress
       ["VBD"; _ref; vbd_operation_to_string (List.hd current_ops)]
       [`unpause] ;
+  (* No hotplug on dom0 *)
+  if Helpers.is_domain_zero ~__context vm then
+    set_errors Api_errors.operation_not_allowed
+      ["Control domain does not support hotplug"]
```

Comment:
Should this be `does not support hot plugging`? Also below.

Reply:
We normally write "hotplug" rather than "hot plug". Regarding "-ing", I'm not convinced...